src/main/webapp/filterScript.js

```diff
@@ -0,0 +1,43 @@
+const UNCLICKED = 'unclicked-filter';
+const CLICKED = 'clicked-filter';
+const CLICKED_ARRAY_URL = 'clicked-array=';
+
+/**
+* Turn a filter a different color when pressed, and change class accordingly.
+**/
+function turnBlueWhenClicked() { //eslint-disable-line
```

Comment:
Add a note for why lint is disabled- e.g. called from an `onclick` attribute in HTML, or to-be used in a future PR.